### ISSUE TITLE
Bluesky client say me I've an "Invalid Handle" on my PDS

### ISSUE THREAD

**Issue by pirmax** [bluesky-social/atproto#1752]
**Describe the bug**

Someone can say why I've an empty profile with "Invalid Handle" ?

**To Reproduce**

1. I created my account from Bluesky Client online, on my PDS.

**Details**

- Operating system: Linux (Ubuntu) & Docker
- Node version: the purpose NodeJS version on my PDS

**Additional context**

https://app.bsky-sandbox.dev/profile/did:plc:hj4rj7yot7w7d33k3ga3cutx

**Comment by TimBurga** [comment]
Did you ever find a resolution for this? I just created my PDS and I have the same issue

**Comment by pirmax** [comment]
> Did you ever find a resolution for this? I just created my PDS and I have the same issue

No resolution for me now.

**Comment by TimBurga** [comment]
This might be helpful: https://bsky-debug.app/handle

**Comment by mai1015** [comment]
I still having this issue as well.. the well-known return unknown user

**Comment by kurtbahartr** [comment]
I just came across a profile that has this same thing. I can access their profile and posts just fine, it's just that I see "Invalid handle" under the name.

Tapping on "Share profile" and copying the link leads me to a link that is what seems to be some kind of an ID for PDS profiles?

Link: `https://bsky.app/profile/did:plc:wqh6[15 more characters]5img`

The link is partially censored as I'm not affiliated with that profile in any way - I just saw one of my friends repost an art from them and this issue got my interest.

**Comment by TimBurga** [comment]
@kurtbahartr bsky was suffering earlier due to the influx of new users. It's possible the invalid handle you saw was related to that. See Paul's post for symptoms: https://bsky.app/profile/pfrazee.com/post/3lawdqryaws2q

**Comment by kurtbahartr** [comment]
> @kurtbahartr bsky was suffering earlier due to the influx of new users. It's possible the invalid handle you saw was related to that. See Paul's post for symptoms: https://bsky.app/profile/pfrazee.com/post/3lawdqryaws2q

@TimBurga, I see. Thanks for letting me know, I missed that post apparently-

**Comment by m00dawg** [comment]
I had this problem and ended up verifying my domain and using that as the handle. Not sure how that might work with multiple accounts but it got things going.

**Comment by mattixittam** [comment]
> I had this problem and ended up verifying my domain and using that as the handle. Not sure how that might work with multiple accounts but it got things going.

Yeah, considering doing the same here. Could this just be a config issue? I followed the [Self-hosting guide](https://atproto.com/guides/self-hosting) to the letter, but am getting the same issue on my first account on the PDS.

**Comment by mai1015** [comment]
I find a way to solve it. So when you register account, you have to make sure that you domain resolve correctly, and the forward is correctly as well if you are using reverse proxy (the API and .well-known). Then, when you register, it should be working. If anything is not configure correctly, the handle will break for the account. And I have to remove the account and re-register. I did not find a way to fix it without removing the account. Here is the config I use for nginx.

<details>

``` nginx
server {
	listen 80;
	listen [::]:80;
	listen 443 ssl http2;
	listen [::]:443 ssl http2;

	server_name example.domain;
	server_name *.example.domain;

	# SSL
	ssl_certificate /etc/nginx/ssl/pub.pem;
	ssl_certificate_key /etc/nginx/ssl/priv.pem;

	# reverse proxy
	location / {
		proxy_pass http://127.0.0.1:9380;
                proxy_set_header Host            $host;
                proxy_set_header X-Forwarded-For $remote_addr;
	}

        location /xrpc/com.atproto.sync.subscribeRepos {
                proxy_pass http://127.0.0.1:9380;
                    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header Host $host;
        }
}
```
</detail>

**Comment by unixfg** [comment]
Does anyone know if there is a particular waiting period?

I had this same problem, but after deleting my bsky account (I saw it disappear from the PDS) I've been unable to recreate with the same handle.

"❌ That handle is already taken."

**Comment by thepeak99** [comment]
What fixed it for me was to first edit my profile and set a name and a picture and then go to settings/account/handle and change my handle to something else and then back to the original.
This only happens if you create the account using the pdsadmin command, if you create an invite and then create the account from the app, it works fine from the beginning.
